src/components/jellyfinApi.ts

```diff
@@ -21,9 +21,9 @@ export abstract class JellyfinApi {
     public static versionNumber = RECEIVERVERSION;
 
     public static setServerInfo(
-        userId: string,
-        accessToken: string,
-        serverAddress: string,
+        userId: string | null,
+        accessToken: string | null,
+        serverAddress: string | null,
```

Comment:
Why not just do something like this? I'm not 100% familiar with default values in typescript so maybe the behavior is different but they seem nicer.

```
userId: string = null
```

Reply:
The behavior is technically different but ultimately achieve the same thing, more or less. the original approach allows null to be passed while the new one would also allow it to be passed, while also defaulting to null. I do see the benefit of having to pass less parameters when only the first or second are needed. I don't have a strong opinion about this so if you feel that the new approach would be better then I'll switch. However it would still have to have null as a possible type, or else the TS compiler will yell at us for assigning null to a string type:

```
userId: string | null = null
```

Reply:
This sounds good enough then. I'm actually just surprised TypeScript doesn't have a better syntax for handling nullability than this to be honest.